docs/04-guides/on-demand/update-video-asset/02-api.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them 
+ready for optimized playback. Videos can be provided either by you (static content) 
+or your users, given your application offers an interface for them to do so. 
+
+You should be familiar with [Assets](/reference/api/get-assets) and [Tasks](/reference/api/get-tasks) 
+and have generated a [Livepeer Studio API key](/quickstart/).
+
+## Step 1: Get the `asset.id` of an existing asset
```

Comment:
We should not advise a developer to list all assets just to find one. There could theoretically be thousands. We should assume they already have the asset id in their own database and are using it to make a request to get more info about it.

Reply:
If that's the case and they already have an `asset.id` wouldn't they just jump to step 2?

Reply:
Yup. You can remove step 1.